visualization/app/codeCharta/state/settingsService/settings.service.events.ts

```diff
@@ -32,6 +32,14 @@ export interface SearchPatternSubscriber {
 	onSearchPatternChanged(searchPattern: string)
 }
 
+export interface FocusNodePathSubscriber {
+	onFocusNodePath(focusedNodePath: string)
+}
+
+export interface UnfocusNodePathSubscriber {
+	onUnfocusNodePath(unfocusNodePath: string)
```

Comment:
an unfocusedNode has no path. Just call it `focusedNodePath`
We could also remove that parameter from this method completly, because we know that whenever this is called, the focusedNodePath will be empty.

Reply:
Yes, I would remove the parameter as well.